<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::ScriptEngineDebuggerWidget</class>
 <widget class="QWidget" name="GammaRay::ScriptEngineDebuggerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QComboBox" name="scriptEngineComboBox"/>
        </item>
       </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
